packages/chrome/src/elements/nav/NavItemText.tsx

```diff
@@ -11,7 +11,7 @@ import { StyledNavItemText, IStyledNavItemTextProps } from '../../styled';
 import { useNavContext } from '../../utils/useNavContext';
 
 /**
- * Accepts all `<span>` attributes and events
+ * @extends HTMLAttributes<HTMLSpanElement>
```

Comment:
The declaration below should be improved to use `& HTMLAttributes<HTMLSpanElement>`